app/css/openmrs-owa-orderentry.scss

```diff
@@ -7,105 +7,130 @@
  * graphic logo is a trademark of OpenMRS Inc.
  */
 
-
 body {
-    font-family: "OpenSans", Arial, sans-serif;
-    -webkit-font-smoothing: subpixel-antialiased;
-    max-width: 1000px;
-    margin: 10px auto;
-    background: #eeeeee;
-    color: #363463;
-    font-size: 16px;
+  font-family: 'OpenSans', Arial, sans-serif;
+  -webkit-font-smoothing: subpixel-antialiased;
+  max-width: 1000px;
+  margin: 10px auto;
+  background: #eeeeee;
+  color: #363463;
+  font-size: 16px;
 }
 
 .body-wrapper {
-    padding-top: 15px;
-    padding-left: 10px;
-    padding-right: 10px;
-    background-color: white;
-    -moz-border-radius: 5px;
-    -webkit-border-radius: 5px;
-    -o-border-radius: 5px;
-    -ms-border-radius: 5px;
-    -khtml-border-radius: 5px;
-    border-radius: 0px 0px 5px 5px;
-    min-height: 500px;
-    justify-content: center;
+  padding-top: 15px;
+  padding-left: 10px;
```

Comment:
Hey, you could use `padding: 15px 10px 0;` here.

Reply:
I know, but this is more readable..